odoo/addons/base/models/res_users.py

```diff
@@ -1700,10 +1708,10 @@ def _generate(self, scope, name):
         # no need to clear the LRU when *adding* a key, only when removing
         k = binascii.hexlify(os.urandom(API_KEY_SIZE)).decode()
         self.env.cr.execute("""
-        INSERT INTO res_users_apikeys (name, user_id, scope, key, index)
+        INSERT INTO {table} (name, user_id, scope, key, index)
         VALUES (%s, %s, %s, %s, %s)
         RETURNING id
-        """,
+        """.format(table=self._table),
         [name, self.env.user.id, scope, hash_api_key(k), k[:INDEX_SIZE]])
 
         ip = request.httprequest.environ['REMOTE_ADDR'] if request else 'n/a'
```

Comment:
Might be worth also using the model description in the logging message (and updating the apikey's which is currently not very useful), so it's not misleading for split-model devices.